ribs/emitters/operators/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""Operators Act as Mutation Functions to archive solutions
```

Comment:
Can you list OperatorBase in this module? You'll need it in both the docstring and in the `__all__` variable.

Also, make the docstring start off with a single summary line, then have a short paragraph with the explanation.

See the ribs.emitters.opt module for an example: https://github.com/icaros-usc/pyribs/blob/master/ribs/emitters/opt/__init__.py